src/components/Header.js

```diff
@@ -36,6 +42,10 @@ class Header extends React.Component {
     })
   }
 
+  handleLogout() {
+    this.location = logoutLink
```

Comment:
This should definitely be `window`

Reply:
I'm getting a linter error when I use `window` instead of `this`. I think that was the reason I used `this` originally... The whole error is this:
` 45:15  error  Expected 'this' to be used by class method 'handleLogout'  class-methods-use-this`